spec.html

```diff
@@ -32,4 +32,94 @@
   }
 </style>
 
-TODO
+<emu-clause id="sec-native-error-types-used-in-this-standard-aggregatenerror">
+  <h1>AggregateError</h1>
+  <p>TODO implement</p>
+</emu-clause>
+
+<emu-intro id="intro">
+  <h1>Introduction</h1>
+  <p>`Promise.any()` accepts an array of promises and returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected with an array of rejection reasons if all of the given promises are rejected.</p>
+</emu-intro>
+
+<emu-clause id="sec-promise.allSettled">
+  <h1>Promise.any ( _iterable_ )</h1>
+  <p>The `any` function returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected with an array of rejection reasons if all of the given promises are rejected. It resolves all elements of the passed iterable to promises as it runs this algorithm.</p>
+  <emu-alg>
+    1. Let _C_ be the *this* value.
+    1. If Type(_C_) is not Object, throw a *TypeError* exception.
+    1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+    1. Let _iteratorRecord_ be GetIterator(_iterable_).
+    1. IfAbruptRejectPromise(_iteratorRecord_, _promiseCapability_).
+    1. Let _result_ be PerformPromiseAny(_iteratorRecord_, _C_, _promiseCapability_).
+    1. If _result_ is an abrupt completion, then
+      1. If _iteratorRecord_.[[Done]] is *false*, set _result_ to IteratorClose(_iteratorRecord_, _result_).
+      1. IfAbruptRejectPromise(_result_, _promiseCapability_).
+    1. Return Completion(_result_).
+  </emu-alg>
+  <p>This function is the <dfn>%Promise_any%</dfn> intrinsic object.</p> <!-- not sure about this -->
```

Comment:
This shouldn’t be necessary now, since nothing else references the function in the spec, and won’t be necessary ever once https://github.com/tc39/ecma262/pull/1376 lands.

Reply:
@ljharb thanks! I just found some other functions there which also don't have another references

Reply:
Some of the existing ones have references in the HTML spec, and so none of them can be safely removed.

Reply:
Makes sense! It'll be handy to have backlinks for such Well-Known Intristic Objects in some annex.